warn/warn_bazel_api.go

```diff
@@ -168,7 +178,12 @@ func notLoadedFunctionUsageCheck(f *build.File, category string, globals []strin
 	walk(&expr, bzlenv.NewEnvironment())
 
 	if fix && len(toLoad) > 0 {
-		f.Stmt = edit.InsertLoad(f.Stmt, loadFrom, toLoad, toLoad)
+		loads := []string{}
+		for k := range toLoad {
+			loads = append(loads, k)
+		}
+		sort.Slice(loads, func(i, j int) bool { return loads[i] < loads[j] })
```

Comment:
You can use `sort.Strings` to sort a slice of strings without providing a comparison function.